assets/js/components/User/register.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="register">
+        <label>Email:</label>
+        <input v-model="email"/>
+        <label>First:</label>
+        <input v-model="first"/>
+        <label>Last:</label>
+        <input v-model="last"/>
+        <button v-on:click="createUser">Register</button>
```

Comment:
If you change `v-on:click` to `@click.prevent`, it will prevent any browser default behavior from interfering with the event

Reply:
Probably not relevant here, I just prefer `@click.prevent` in general